src/views/TradingCompetition/index.tsx

```diff
@@ -99,8 +99,11 @@ const TradingCompetition = () => {
   const [team3LeaderboardInformation, setTeam3LeaderboardInformation] = useState({ teamId: 3, leaderboardData: null })
 
   const isCompetitionLive = currentPhase.state === LIVE
-  const hasCompetitionEnded =
-    currentPhase.state === FINISHED || currentPhase.state === CLAIM || currentPhase.state === OVER
+  const hasCompetitionEnded = [
+    FINISHED,
+    CLAIM,
+    OVER
+  ].some((phasesItem) => phasesItem === currentPhase.state)
```

Comment:
In this case I think it is better to use `includes`
```
const hasCompetitionEnded = [
  FINISHED,
  CLAIM,
  OVER
].includes(currentPhase.state)
```

Reply:
Yeah, It was much better 👍

Reply:
`
export const EndedPhasesSet = new Set([
  FINISHED,
  CLAIM,
  OVER
])
`

In this case, Set may better than Array.
So I just try modify the constant file for a new set, then import by "TradingCompetition" to use it.

d68109852c96cc46d309396f0e54c258c9274505